exporter/kafkaexporter/config.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kafkaexporter
+
+import "go.opentelemetry.io/collector/config/configmodels"
+
+// Config defines configuration for Kafka exporter.
+type Config struct {
+	configmodels.ExporterSettings `mapstructure:",squash"`
+	// The list of kafka brokers (default localhost:9092)
+	Brokers []string `mapstructure:"brokers"`
```

Comment:
Kafka client accepts a list of brokers not only a single address. In statically configured environments this can be handy.

It is the initial list of broker addresses that is used to discover all nodes  https://docs.confluent.io/current/installation/configuration/consumer-configs.html#bootstrap.servers